### ISSUE TITLE
"x - +" buttons overlap with Messenger settings

### ISSUE THREAD

**Issue by dhharris** [rsms/fb-mac-messenger#387]
To illustrate, here is the problem. I've been able to reproduce it by closing the app and reopening. Happens at random it seems.

 ![image](http://i.imgur.com/LHnyGoO.png)

**Comment by karteek-paruchuri** [comment]
Yeah this exists. But to by pass it temporarily, do a CMD + R and it disappears.

**Comment by dhharris** [comment]
Yeah, it's not a big deal, just letting you guys know that bug is there.

**Comment by simon-ondrej** [comment]
Reporting the same problem. Not a big inconvenience either. Just letting you know.

**Comment by rsms** [comment]
Known issue. Very long story, but the really short version is: Facebook keeps changing the HTML and JS of messenger.com meaning that this happens very often. We've tried to fix this numerous times during the last year or two, but at this point I'm not even going to try to fix it. The current code does a "best guess attempt" to find and hide the menu, but sometimes it doesn't succeed. Try reloading (cmd-R) and you might get lucky.

If someone wants to spend their time fixing this, feel free to send a PR and remember that you absolutely can not rely on CSS class names as they change as often as weekly (auto-generated.)

**Comment by chrismessina** [comment]
FYI, there's another case where this occurs when viewing someone's Messenger Day.

![image](https://cloud.githubusercontent.com/assets/22610/25816255/5cc726a2-33d8-11e7-8593-01712ddad4f0.png)

**Comment by andrewmccullough** [comment]
![screen shot 2017-06-04 at 6 15 32 pm](https://cloud.githubusercontent.com/assets/6857304/26765862/dfba5fc2-4952-11e7-9045-30bcee55d532.png)

Still an issue! Just FYI...
